### DISCUSSION TITLE
Help with dropdown link on Bootstrap 5.3

### DISCUSSION THREAD

**Discussion by dafaher** [twbs/bootstrap#40518] [Q&A] [answered]
Good evening community, I'm sorry to bother you. I'm using a base navbar with a dropdown link, and I need one of those links to show me two more links. I've been trying to get it to work for days but I can't (I'm using bootstrap 5.3 and I'm new to this, so I get confused easily), the documentation doesn't show how to do exactly this. This is my current code:

```html
<div class="collapse navbar-collapse" id="navbarSupportedContent">
  <ul class="navbar-nav mx-auto mb-2 mb-lg-0">
    <li class="nav-item">
      <a class="nav-link active" aria-current="page" href="index.html">Home</a>
    </li>
    <li class="nav-item">
      <a class="nav-link" href="about-us.html">About us</a>
    </li>
    <li class="nav-item dropdown">
      <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
      Services
      </a>
      <ul class="dropdown-menu">
        <li><a class="dropdown-item" href="#">Action</a></li>
        **<li><a class="dropdown-item" href="#">Another action</a></li>**
        <li><hr class="dropdown-divider"></li>
        <li><a class="dropdown-item" href="#">Something else here</a></li>
      </ul>
    </li>
    <li class="nav-item">
      <a class="nav-link" href="news.html">News</a>
    </li>
    <li class="nav-item">
      <a class="nav-link" href="contact.html">Contact</a>
    </li>
  </ul>
  <form class="d-flex" role="search">
    <input class="form-control me-2" type="search" placeholder="Búsqueda" aria-label="Search">
    <button class="btn btn-outline-success" type="submit">Búsqueda</button>
  </form>
  </ul>
</div>
```

Where it says "**Another action**" I need 2 other links to be shown on the right, as a tree. Any idea on how to do that? Everything I tried failed. It seems so simple, and yet so tricky. Thanks in advance for any suggestion you can share!

---

_Edited by maintainer for syntax highlighting and code formatting_

**Comment by julien-deramond** [comment]
Hi there, it's not an answer to your question, but just for information (as it may break other things), you have 2 opening `<ul>` and 3 closing `</ul>` in your example.

**Reply by dafaher** [reply]
Thank you very much anyway, you have a good eye haha! I was trying different things to see if I could figure out what I need on my own, and apparently I left that line. Thank you!

**Comment by dafaher** [comment, marked as answer]
In case it helps anyone, after days of research I found the solution in this link:

**[https://mdbootstrap.com/docs/standard/extended/dropdown-multilevel/#](https://mdbootstrap.com/docs/standard/extended/dropdown-multilevel/#)**

You can find the necessary html+css code in the "Navbar dropdown" section. It worked like a charm. Thanks everyone for reading!
